tests/semanticdb/metac.expect

```diff
@@ -2184,87 +2193,91 @@ Schema => SemanticDB v4
-example/Methods#m18. => val method m18
-example/Methods#m19$default$2(). => method m19$default$2
-example/Methods#m19$default$3(). => method m19$default$3
-example/Methods#m19$default$3().(x) => param x
-example/Methods#m19$default$3().(y) => param y
-example/Methods#m19(). => method m19
-example/Methods#m19().(x) => param x
-example/Methods#m19().(y) => param y
-example/Methods#m19().(z) => param z
-example/Methods#m20(). => method m20
-example/Methods#m20().(a) => param a
-example/Methods#m20(+1). => method m20
-example/Methods#m20(+1).(b) => param b
-example/Methods#m20(+2). => var method m20
-local0 => abstract val method x
+example/Methods# => class Methods [typeparam T ] extends Object { self: Methods[T] => +44 decls }
+example/Methods#AList# => type AList [typeparam T ] = List[T]
+example/Methods#AList#[T] => typeparam T 
+example/Methods#List# => class List [typeparam T ] extends Object { self: List[T] => +2 decls }
+example/Methods#List#[T] => typeparam T 
+example/Methods#List#`<init>`(). => primary ctor <init> [unknown T: <?>]: List[T]
+example/Methods#[T] => typeparam T 
+example/Methods#`<init>`(). => primary ctor <init> [unknown T: <?>]: Methods[T]
+example/Methods#`m8().`(). => method m8(). (): Nothing
+example/Methods#`m9().`# => class m9(). extends Object { self: m9(). => +1 decls }
+example/Methods#`m9().`#`<init>`(). => primary ctor <init> (): m9().
+example/Methods#`m20_=`(). => var method m20_= (param x$1: <?>.m17.type): Unit
+example/Methods#`m20_=`().(x$1) => param x$1: <?>.m17.type
+example/Methods#m1(). => method m1 => Nothing
+example/Methods#m2(). => method m2 (): Nothing
+example/Methods#m3(). => method m3 (param x: Int): Nothing
+example/Methods#m3().(x) => param x: Int
+example/Methods#m4(). => method m4 (param x: Int)(param y: Int): <?>
+example/Methods#m4().(x) => param x: Int
+example/Methods#m4().(y) => param y: Int
+example/Methods#m5(). => method m5 (param x: String): Nothing
+example/Methods#m5().(x) => param x: String
+example/Methods#m5(+1). => method m5 (param x: Int): Nothing
+example/Methods#m5(+1).(x) => param x: Int
+example/Methods#m6(). => method m6 (param x: Int): Nothing
+example/Methods#m6().(x) => param x: Int
+example/Methods#m6(+1). => method m6 (param x: List[T]): Nothing
+example/Methods#m6(+1).(x) => param x: List[T]
+example/Methods#m6(+2). => method m6 (param x: List[T]): Nothing
+example/Methods#m6(+2).(x) => param x: List[T]
+example/Methods#m7(). => method m7 [typeparam U ](param c: Methods[T], param l: List[U])(unknown evidence$1: <?>): <?>
+example/Methods#m7().(c) => param c: Methods[T]
+example/Methods#m7().(l) => param l: List[U]
+example/Methods#m7().[U] => typeparam U 
+example/Methods#m9(). => method m9 (param x: m9().): Nothing
+example/Methods#m9().(x) => param x: m9().
+example/Methods#m10(). => method m10 (param x: List[T]): Nothing
+example/Methods#m10().(x) => param x: List[T]
+example/Methods#m11(). => method m11 (param x: <?>.Predef.type): Nothing
```

Comment:
it seems like singleton types are not being built using their prefix yet

Reply:
although I looked at `TypeOps` and this seems correct, and comparing to metac.expect in the scalameta repo, I guess it is just an issue with formatting